bot/plugins/instance.py

```diff
@@ -30,12 +30,15 @@ class ComponentID(enum.StrEnum):
     INSTRUCTION_SET = "instruction_set"
     COMPILER_TYPE = "compiler_type"
     VERSION = "version"
-    STDIN = "stdin"
+    INPUTS = "inputs"
 
 
 class ModalID(enum.StrEnum):
     LANGUAGE = "language"
+    INPUTS = "inputs"
     STDIN = "stdin"
+    COMPTIME = "comptime"
+    RUNTIME = "runtime"
```

Comment:
These are unused I think?

Reply:
They aren't

Reply:
![image](https://github.com/CircuitSacul/io/assets/65515165/75940fbe-38bd-4288-836e-f12bacaa1b74)

I am using them here.

Reply:
Line 230

Reply:
Can you clarify this one?